src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_xml.py

```diff
@@ -91,6 +91,12 @@ def _create_default_type_map():
     except:
         pass
 
+    try:
+        from ctypes import Array
```

Comment:
Seems good to me, but it'd be nice to have a test too...

Reply:
I see what you mean (it tests the resolver themselves, not the resolution process), but I think that using `test_resolvers` for that is also a good place (the API to be used is `get_type(obj)` where the last item is the actual resolver... there's only one test there that calls this API, but I think we can extend that same module to add this test).